bin/ecs-deploy.sh

```diff
@@ -0,0 +1,2 @@
+#! bin/bash
```

Comment:
Will do

Reply:
I didn't re-enable the call to bin/ecs-deploy.sh because I don't have a copy in the repo. I looked at the example you want to copy from, but it looks large and complex so I would feel more comfortable getting everything else working besides it and then spend some time looking at the script to get more comfortable before adding it myself.

Alternatively someone more comfortable in that area could add it.

Reply:
OK, understood, and I can agree with your assessment.  Since this isn't crucial for the current goal of "getting container images to publish to ECR", I'll add this to a separate issue and see about PR'ing it before we show up tomorrow morning (crossing fingers).

Reply:
I added the call to bin/ecs-deploy.sh back in to deploy.sh and added a blank ecs-deplpy.sh. Should demonstrate the ability to call and leave it ready to be filled-in.